keyboards/omnikeyish/config.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "config_common.h"
+
+#define KEYBOARD_PCB_REV 11
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0666
+#define DEVICE_VER      0x1337
+#define MANUFACTURER    Henrik O. Sørensen
+#define PRODUCT         Omnikey(-ish) Keyboard
+#define DESCRIPTION     Replacement PCB for Omnikey keyboards
+
+/* key matrix size */
+#define MATRIX_ROWS 6
+#define MATRIX_COLS 23
+
+/* key matrix pins */
+#if KEYBOARD_PCB_REV == 10
```

Comment:
Is an older revision in the wild?

It might be worth setting up a `rev1` `rev2` folder setup. While slightly more complicated, it allows users to select the appropriate revision (including in the Configurator) without editing a header.

Reply:
No. Rev 1.0 was never published on github. And it consists of all of 5 PCBs I ordered from jlcpcb.
Seems unlikely anyone else will ever build one. 

Only two changes between 1.0 and 1.1:
Discovered that unlike every other measurement, the fblock distance was metric (50mm, not 50.8mm (2"). So the origina switchl plate didn't QUITE fit the 1.0 pcb. Added an extra set of pads.
Teensy++ 2.0 hijacks PD6 for an LED, so that played havoc with the fifth row, until I looked at the teensy schematic and headpalmed. Moved row 5 to PE6 instead on rev 1.1.